enhancements/update/targeted-update-edge-blocking.md

```diff
@@ -0,0 +1,247 @@
+    or
+    max by (_id) (subscription_labels{managed="false"})
+```
+
+The query returns `{_id="..."}=0` results for `managed="true"` clusters (blocking the edge) and `{_id="..."}=1` results for `managed="false"` clusters (allowing the edge).
+Clusters with both `managed="true"` and `managed="false"` records in the current time window, because a cluster transitioned into or out of the managed state, will fall into the safer "block" bucket (the `or` semantics are defined [here][PromQL-or]).
+
+### Risks and Mitigations
+
+#### Divergent Prometheus services
+
+There is a risk that [published graph-data information](distribute-secondary-metadata-as-container-image.md) which assumes access to Red Hat's internal Telemetry will block edges when consumed by update services without access to the internal Telemetry.
+The risk is small initially, when we have no such edges.
+If targeted blocking becomes widespread, a simple but tedious mitigation would be asking users to maintain their own graph-data (e.g. removing all targeted edge-blocking references) and build their own graph-data image.
+
+A more powerful but complicated fix would be documenting a process for running a local Telemetry aggregator and configuring the local update service to run queries against that aggregator.
+This documentation would need to cover mechanisms for adding any expected metrics that Red Hat currently adds in internal Telemetry which come up in targeted edge queries, such as [`subscription_labels`](#openshift-dedicated).
+
+Users who did not want to maintain their own graph-data or a local Telemetry aggregator could also deploy their local update service with the targeted edge blocking plugin disabled.
+This will leave all conditional edges *enabled*, with the local administrators taking responsibility for otherwise blocking or avoiding conditional edges to which their client cluster set may be vulnerable.
+
+#### Clusters not reporting data
+
+Clusters which have [opted out of uploading Telemetry][uploaded-telemetry-opt-out] or which run on restricted networks and so are unable to report Telemetry will default to the blocked state (more discussion on unknown ID handling in [the *query coverage* section](#query-coverage)).
+Being unable to report Telemetry while still being able to connect to Red Hat's hosted update service seems unlikely; if a cluster can do one (e.g. via a proxy), it should be able to do both.
+Clusters which connect to neither Telemetry nor Red Hat's hosted update service may run a local update service, and that is discussed in [the *divergent Prometheus services* section](#divergent-prometheus-services).
+
+Clusters which opt out of Telemetry are likely still able to connect to the update service.
+When they do, they will be [blocked as cache-misses](#enhanced-graph-data-schema-for-blocking-edges).
+I think that conservative approach is acceptable, because we cannot tell if the non-reporting cluster is vulnerable to the known issues or not.
```

Comment:
Who needs to sign off on this vs. the possibility of breaking affected clusters because they weren't reporting Telemetry and the update service sid "maybe ok?" and recommended the edge?